src/crimson/os/seastore/btree/fixed_kv_btree.h

```diff
@@ -1293,6 +1293,8 @@ class FixedKVBtree {
       ceph_assert(depth == meta.depth);
       ceph_assert(begin == meta.begin);
       ceph_assert(end == meta.end);
+      ceph_assert(ret->get_in_extent_checksum()
+        == ret->get_last_committed_crc());
```

Comment:
Checksum mismatch may not be logical bugs -- it may caused by device issues -- so print mismatched information in error log is necessary here.

Also the checksum validation should be moved before all the other validations.